### ISSUE TITLE
Blinkist metadata

### ISSUE THREAD

**Issue by hcbille** [jo1gi/audiobook-dl#49]
**Installation method:**
Github repo

**Version:**
latest version pulled from github to get blinkist.com support

**Describe the bug**
The metadata does not correctly add the author.
<img width="407" alt="image" src="https://github.com/jo1gi/audiobook-dl/assets/32437363/c2311827-cd43-4892-86c3-a701b8376d22">
This is after downloading my books.

Metadata for a file:
```
kMDItemAlternateNames                  = (
    "12 Rules For Life - Part 0.m4a"
)
kMDItemAudioBitRate                    = 128155
kMDItemAudioChannelCount               = 1
kMDItemAudioSampleRate                 = 44100
kMDItemAuthors                         = (
    n
)
kMDItemContentCreationDate             = 2023-08-07 15:23:30 +0000
kMDItemContentCreationDate_Ranking     = 2023-08-07 00:00:00 +0000
kMDItemContentModificationDate         = 2023-08-07 15:28:08 +0000
kMDItemContentModificationDate_Ranking = 2023-08-07 00:00:00 +0000
kMDItemContentType                     = "com.apple.m4a-audio"
kMDItemContentTypeTree                 = (
    "com.apple.m4a-audio",
    "public.mpeg-4-audio",
    "public.audio",
    "public.audiovisual-content",
    "public.data",
    "public.item",
    "public.content"
)
kMDItemDateAdded                       = 2023-08-07 15:23:30 +0000
kMDItemDateAdded_Ranking               = 2023-08-07 00:00:00 +0000
kMDItemDisplayName                     = "12 Rules For Life"
kMDItemDocumentIdentifier              = 0
kMDItemDurationSeconds                 = 73.02428571428571
kMDItemFSContentChangeDate             = 2023-08-07 15:28:08 +0000
kMDItemFSCreationDate                  = 2023-08-07 15:23:30 +0000
kMDItemFSCreatorCode                   = ""
kMDItemFSFinderFlags                   = 0
kMDItemFSHasCustomIcon                 = (null)
kMDItemFSInvisible                     = 0
kMDItemFSIsExtensionHidden             = 0
kMDItemFSIsStationery                  = (null)
kMDItemFSLabel                         = 0
kMDItemFSName                          = "12 Rules For Life - Part 0.m4a"
kMDItemFSNodeCount                     = (null)
kMDItemFSOwnerGroupID                  = 20
kMDItemFSOwnerUserID                   = 501
kMDItemFSSize                          = 1189243
kMDItemFSTypeCode                      = ""
kMDItemInterestingDate_Ranking         = 2023-08-07 00:00:00 +0000
kMDItemKind                            = "Apple MPEG-4 audio"
kMDItemLogicalSize                     = 1189243
kMDItemMediaTypes                      = (
    Sound
)
kMDItemPhysicalSize                    = 2236416
kMDItemTitle                           = "12 Rules For Life"
kMDItemTotalBitRate                    = 128155
```

**Comment by jo1gi** [comment]
Should be fixed in 17651ff.

**Comment by hcbille** [comment]
Fixed by the above! :D
